admin/common_auth/views.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth import authenticate, logout as logout_user, login as auth_login
+from django.shortcuts import render, redirect
+
+from forms import LoginForm
+
+def login(request):
+    if request.user.is_authenticated():
+        return redirect('/admin/')
```

Comment:
Better yet, there are [login/logout views](https://docs.djangoproject.com/en/1.8/topics/auth/default/#django.contrib.auth.views.login) already built into `django.contrib.auth.views`. You could just reference those in your urls file until we need to do something they don't do.
```python
from django.contrib.auth import views as auth_views
url(r'^login/?$', auth_views.login, {'template_name': 'login.html'}),
```